src/system/Zikula/Module/MailerModule/MailerModuleInstaller.php

```diff
@@ -92,6 +95,8 @@ public function upgrade($oldversion)
                 unset($config['spool']);
                 $configDumper->setConfiguration('swiftmailer', $config);
             case '1.4.1':
+                HookUtil::registerSubscriberBundles($this->version->getHookSubscriberBundles());
+            case '1.4.2':
```

Comment:
in the other PR, the hooks are registered in 1.3.2 - adding them here is confusing because the versions would differ. I'm not sure of a better solution however. @Guite @cmfcmf your opinion?